src/containers/app.js

```diff
@@ -18,6 +21,8 @@ class App extends React.Component {
 
   componentDidMount() {
     window.addEventListener('scroll', this.handleScroll);
+    ReactGA.initialize(CONST.GA_ID);
```

Comment:
Shouldn't we move the initialisation to the `index.js` file and also use the `withTracker` function with this component?